Content.Client/Entry/EntryPoint.cs

```diff
@@ -91,7 +98,9 @@ public override void Init()
             _prototypeManager.RegisterIgnore("utilityCurvePreset");
             _prototypeManager.RegisterIgnore("accent");
             _prototypeManager.RegisterIgnore("material");
-            _prototypeManager.RegisterIgnore("reaction"); //Chemical reactions only needed by server. Reactions checks are server-side.
+            _prototypeManager
+                .RegisterIgnore(
+                    "reaction"); //Chemical reactions only needed by server. Reactions checks are server-side.
```

Comment:
crying

Reply:
This is why you don't do comments at the end of a line.